### DISCUSSION TITLE
Renaming "Execution Core Container" in "Connector Core Service(s)"

### DISCUSSION THREAD

**Discussion by HeinrichPet** [International-Data-Spaces-Association/IDS-RAM_4_0#83] [Ideas]
The name "Execution Core Container" strongly suggests that there is exactly one container that provides the important IDS functionalities. In the meantime, however, there are also some implementations that split the IDS Connector into several services instead of one container - keyword microservice architecture. To pay tribute to this fact, I suggest in discussion with Monika and Gerd that we choose a more appropriate name.

Our suggestion would be e.g. "**Connector Core Service(s)**". This should pick up the "core" idea of the Execution Core Container as well as the "service" idea of microservices. So "Connector Core Service(s)" can be exactly one monolithic software stack, or a multitude of services that together make up the connector.

It is important that the connector contains at least the functionalities that are shown in the following image: [Functional View](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/media/3_5_system_layer/connector_functional_view.jpg)
A descriptive text follows in the system layer of RAM 4.0.

Are there any opinions on that?

**Comment by mokamhuber** [comment]
Sounds good to me :)
